sdk/opa.go

```diff
@@ -510,6 +510,8 @@ type evalArgs struct {
 	txn                 storage.Transaction
 	queryCache          *queryCache
 	interQueryCache     cache.InterQueryCache
+	regexBuiltinCache   cache.ValueCache
+	globBuiltinCache    cache.ValueCache
```

Comment:
Should these also be added to the `sdk.OPA.state`, like `InterQueryCache`?

Reply:
Yes, will update.